main.go

```diff
@@ -4,18 +4,15 @@ import (
 	"log"
 	"os"
 
-	"github.com/xlab/closer"
-
 	"github.com/ivanterekh/go-skeleton/server"
 )
 
 func main() {
-	port := os.Getenv("PORT")
-	if port == "" {
-		port = "8080"
+	address := os.Getenv("ADDRESS")
```

Comment:
Should be encapsulated somehow. I'd recommend to create your own function. For example, you can create functions `env.GetString("NAME", "DEFAULT VALUE") string`, `env.GetInt("NAME", 42) int` and etc.